web/src/components/RasaWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement, useEffect } from 'react'
+import { useTheme } from 'styled-components'
+
+const RasaWidget = (): ReactElement => {
+  const theme = useTheme()
+
+  useEffect(() => {
+    const script = document.createElement('script')
+
+    script.src = 'https://unpkg.com/@rasahq/rasa-chat'
+    script.async = false
```

Comment:
No async and no defer now?

Reply:
Hmh, what was I doing there. :thinking: Will fix.